graph/graph_unix.go

```diff
@@ -0,0 +1,8 @@
+// +build !windows
+
+package graph
+
+// allowBaseParentImage allows images to define a custom parent that is not
+// transported with push/pull but already included with the installation.
+// Only used in Windows.
+const allowBaseParentImage = false
```

Comment:
I wonder if this file should not be called `graph_unix.go` since it's actually built on all non-Windows platforms. It's not a major distinction right now, but could become an issue in the future.

Maybe a way to avoid the two new files and duplication of this variable declaration would be to define `var allowBaseParentImage = false` in `pull_v2.go`, and have a `pull_v2_windows.go` with an `init` function that overrides it.